config/development.yml

```diff
@@ -17,3 +17,4 @@ public:
   default_otp:
     enabled: true
     code: '1234'
+    whitelist: ['9999999999', '8888888888']
```

Comment:
Remove `whitelist`.

I think in non prod, we are already sending default OTP, right? So this should not be needed.